polyA/constants.py

```diff
@@ -54,6 +54,12 @@
 value doesn't matter, but it's here for clarity.
 """
 
+CROSS_MATCH_ADJUSTMENT = 0.999
+"""
+A constant value used in the equations from cross_match to calculate 
+the complexity adjusted score of an alignment.
+"""
```

Comment:
Why this particular value? Why not `0.9` or `0.9999`?

Reply:
I can't find a clear reason on why 0.999 is added, this is just the value given in the equations for how cross match performs complexity adjusted scoring

Reply:
We should explicitly state that it's just the value cross match uses, then. The current comment sort of does that, but it's not entirely clear, at least to me.